gravitee-apim-rest-api/gravitee-apim-rest-api-services/gravitee-apim-rest-api-services-dynamic-properties/src/main/java/io/gravitee/rest/api/services/dynamicproperties/DynamicPropertiesService.java

```diff
@@ -96,17 +99,31 @@ public void onEvent(Event<ApiEvent, ApiEntity> event) {
 
         switch (event.type()) {
             case DEPLOY:
-                startDynamicProperties(api);
+                if (hasServiceEnabled(api)) {
```

Comment:
I think it's not necessary because the same test is already done in the `startDynamicProperties`

Reply:
✅